examples/hif2a/fit_to_multiple_rbfes.py

```diff
@@ -160,6 +160,11 @@ def _save_forcefield(fname, ff_params):
         client = GRPCClient(hosts=args.hosts)
     client.verify()
 
+    # create path if it doesn't exist
+    output_path = Path(args.output_path)
```

Comment:
Moved this so that if any of the validation fails, it doesn't create the directory